### PR TITLE
Hide keyboard

### PR DESCRIPTION
**Pull request by vishwesh3**
Hide keyboard on narrowing to searched stream/person private message.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@vishwesh3, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 1]
```diff
@@ -542,6 +524,20 @@ public Cursor runQuery(CharSequence charSequence) {
         }
     }
 
+    private void resetPeopleSearch() {
```
Comment (Sam1301) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Although the function names are self-explanatory, it'd be nice to have a javadoc comment (documentation like this really helps beginners to our project)

**Comment by Sam1301** [comment]
@vishwesh3 Thanks for working on this. The commit message should start with a capital letter (consider having a look at this https://zulip.readthedocs.io/en/latest/version-control.html#commit-messages).

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 2]
```diff
@@ -1024,6 +1020,7 @@ public boolean onChildClick(ExpandableListView parent, View v, int groupPosition
 
             @Override
             public boolean onGroupClick(ExpandableListView expandableListView, View view, int position, long l) {
+                resetStreamSearch();
```
Comment (Sam1301) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Move this line to https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/activities/ZulipActivity.java#L1055.

**Comment by vishwesh3** [comment]
@Sam1301 Done with the changes 
videotogif_2016 12 29_16 07 11

**Comment by Sam1301** [comment]
The code looks fine to me :) Thanks for working on this @vishwesh3 .

**Comment by timabbott** [comment]
@vishwesh3 can you clean up your commit message?  
Check out the Zulip commit message guidelines for more details: https://zulip.readthedocs.io/en/latest/version-control.html#commit-messages

**Comment by vishwesh3** [comment]
@timabbott changed commit message. Hope this would be good one. :)

**Comment by kunall17** [comment]
@vishwesh commit is changing the cursor as well so maybe rename..:) 
@timabbott LGTM! :) 👍

**Comment by timabbott** [comment]
Merged, thanks @vishwesh3 and reviewers!!

**Pull request closed without merging**